Sources/FluentUI_iOS/Components/Command Bar/CommandBarItem.swift

```diff
@@ -5,7 +5,22 @@
 
 import UIKit
 
-public typealias CommandBarItemGroup = [CommandBarItem]
+@objc(MSFCommandBarItemGroup)
+public class CommandBarItemGroup: NSObject {
+    public let items: [CommandBarItem]
+    public let label: String?
+
+    @objc public init(items: [CommandBarItem]) {
```

Comment:
I don't have a *strong* preference, but how do you feel about making this a suppressed argument name with an underscore?
```swift
    @objc public init(_ items: [CommandBarItem]) {
        // ...
    }

    @objc public init(_ items: [CommandBarItem], label: String? = nil) {
        // ...
    }
```